Directory.Build.targets

```diff
@@ -3,10 +3,10 @@
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'
-                          and $(MicrosoftNETCoreAppPackageVersion.StartsWith('$(_TargetFrameworkVersionWithoutV)'))">
+                          and $(MicrosoftNETCoreAppRuntimewinx64PackageVersion.StartsWith('$(_TargetFrameworkVersionWithoutV)'))">
     <FrameworkReference
         Update="Microsoft.NETCore.App" 
-        TargetingPackVersion="$(MicrosoftNETCoreAppPackageVersion)" 
-        RuntimeFrameworkVersion="$(MicrosoftNETCoreAppPackageVersion)" />
+        TargetingPackVersion="$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)"
+        RuntimeFrameworkVersion="$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)" />
```

Comment:
This is questionable and should be a separate dependency on Ref package. But existing.

Reply:
Yes. It was one of the things we rushed to get builds out the door.

Reply:
The diff before has the same problem just harder to spot, so this is a step forward!

Reply:
It's one half of the fix :)